release-notes/5.10.4.md

```diff
@@ -0,0 +1,58 @@
+# CiviCRM 5.10.4
+
+Released February 22, 2019
+
+- **[Synopsis](#synopsis)**
+- **[Bugs resolved](#bugs)**
+- **[Credits](#credits)**
+- **[Feedback](#feedback)**
+
+## <a name="synopsis"></a>Synopsis
+
+| *Does this version...?*                                         |         |
+|:--------------------------------------------------------------- |:-------:|
+| Fix security vulnerabilities?                                   |   no    |
+| Change the database schema?                                     |   no    |
+| Alter the API?                                                  |   no    |
+| Require attention to configuration options?                     |   no    |
+| Fix problems installing or upgrading to a previous version?     |   no    |
+| Introduce features?                                             |   no    |
+| **Fix bugs?**                                                   | **yes** |
+
+## <a name="bugs"></a>Bugs resolved
+
+- **([dev/core#747](https://lab.civicrm.org/dev/core/issues/747)) Cannot
+  load contact images on contact screen following security release
+  ([13679](https://github.com/civicrm/civicrm-core/pull/13679))**
+
+  Following the security release, loading of images on the contact summary
+  screen broke due to the removal of handling of the filename url parameter
+
+- **([dev/report#10](https://lab.civicrm.org/dev/report/issues/10)) Pagination
+  doesn't work on the contribution detail report
+  ([13677](https://github.com/civicrm/civicrm-core/pull/13677))**
+
+  Fixed a recent regression where the pagination on the contribution detail report
+  broke.
+
+- **([dev/core#746](https://lab.civicrm.org/dev/core/issues/746)) Search Builder - Fix
+  regression when > 1 smart group where clauses were used
+  ([13667](https://github.com/civicrm/civicrm-core/pull/13667))**
+
+  Fixed a recent regression whereby if you had > 1 where clauses involving groups
+  and the groups were smart groups it wouldn't return the correct results
+
+
+## <a name="credits"></a>Credits
+
+This release was developed by the following authors and reviewers:
+
+Wikimedia Foundation - Eileen McNaughton; Fuzion - Jitendra Purohit;
```

Comment:
Jon G should be in here - the contribution report one

Reply:
added